implementations/rust/ockam/ockam_api/src/nodes/service/manager.rs

```diff
@@ -657,23 +671,39 @@ pub struct ApiTransport {
     pub flow_control_id: FlowControlId,
 }
 
+#[derive(Debug)]
+pub struct NodeManagerTransportOptionsS<T> {
```

Comment:
Maybe this could be renamed to something simpler like `NodeManagerTransport`

Reply:
Nice catch, I need to rename it